runtime/bcverify/bcverify.c

```diff
@@ -2404,15 +2397,20 @@ j9bcv_verifyBytecodes (J9PortLibrary * portLib, J9Class * clazz, J9ROMClass * ro
 
 	romMethod = (J9ROMMethod *) J9ROMCLASS_ROMMETHODS(romClass);
 
-	if (verboseVerification) {
+	if (verboseVerification && !isNPEMsg) {
```

Comment:
Do not use `!` on non-booleans - do the compare properly please, or create a boolean local.

Reply:
Nevermind, I can't read.